plugins/tedge_apt_plugin/Cargo.toml

```
@@ -27,7 +27,9 @@ clap = { version = "3", features = ["derive"] }
 csv = "1.1"
 regex = "1"
 serde = { version = "1", features = ["derive"] }
+tedge_config = { path = "../../crates/common/tedge_config" }
 thiserror = "1.0"
+toml = "0.5.11"
```

We decided to not specify the PATCH number of the dependencies.

```suggestion
toml = "0.5"
```